client/src/views/Home.vue

```diff
@@ -36,8 +37,8 @@
 
 <script>
 import Debug from 'debug';
+import Top from '../components/top.vue';
 import HaloModelForm from '../components/HaloModelForm.vue';
-import CosmologyForm from '../components/CosmologyForm.vue';
```

Comment:
Why was this removed? Just tried it with it included and it still works fine.

Reply:
Conflict resolution was a little messy! My bad. Good reason to add tests haha

Reply:
Haha alright cool. No worries 🎸